examples_tests/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ add_subdirectory(09.ColorSpaceTest EXCLUDE_FROM_ALL)
 add_subdirectory(10.AllocPerfTest EXCLUDE_FROM_ALL)
 add_subdirectory(11.RayCastCollision EXCLUDE_FROM_ALL)
 add_subdirectory(12.Splines EXCLUDE_FROM_ALL)
+add_subdirectory(13.BRDF_Explorer EXCLUDE_FROM_ALL)
```

Comment:
only if irrbaw is configured to be built with CEGUI

Reply:
updated.